push/index.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: basic
+title: AeroGear Push
+---
+
+# AeroGear Push
+
+AeroGear has support for different push notification technologies:
+
+* [UnifiedPush Server](#unifiedpush-server)
```

Comment:
Perhaps we may note the platform that Unified Push support in a parenthesis:

* UnifiedPush (supports Android, iOS, Windows, FirefoxOS)

I know it is noted right bellow, but still may help people navigate easily - sometimes you just want "Push to Android", and term UnifiedPush is far from your thinking